bundler/lib/bundler/man/bundle-config.1.ronn

```diff
@@ -170,8 +105,7 @@ learn more about their operation in [bundle install(1)](bundle-install.1.html).
    Whether a `bundle install` without an explicit `--path` argument defaults
    to installing gems in `.bundle`.
 * `deployment` (`BUNDLE_DEPLOYMENT`):
-   Disallow changes to the `Gemfile`. When the `Gemfile` is changed and the
-   lockfile has not been updated, running Bundler commands will be blocked.
+   Equivalent to: `BUNDLE_FROZEN`=true, `BUNDLE_PATH`=vendor/bundle.
```

Comment:
Since Bundler can be configured in more ways that environment variables, I'd use


```suggestion
   Equivalent to setting `frozen` to `true` and `path` to `vendor/bundle`.
```